gui/src/components/dashboard/cortx-dashboard-alert-card.vue

```diff
@@ -66,31 +75,31 @@ export default class CortxDashboardAlertCard extends Mixins(AlertsMixin) {
     return [
       {
         title: this.fatalCount,
-        description: "Fatal",
+        description: this.$t("dashboard.fatal"),
```

Comment:
can you move this.$t() into the cortx-dashboard-info-card so that any description passed is directly localized.

Reply:
Done